doc/api/vm.md

```diff
@@ -437,6 +437,9 @@ changes:
     `cachedData` property of the returned `vm.Script` instance.
     The `cachedDataProduced` value will be set to either `true` or `false`
     depending on whether code cache data is produced successfully.
+  * `resolveDynamicImport` {Function} See [`module.link()`][]
+    * `specifier` {string}
+    * `scriptOrModule` {vm.Script|vm.Module}
```

Comment:
We should probably note the return value expected here - `vm.Module | ModuleNamespace`.

Reply:
Out of interest, what is the default context for a `vm.Module` instance? I'm just wondering if a module were to be a thenable to a `vm.Module` instance if that would enable the sandbox code to break out into the non-sandbox context?

Reply:
default context is the context it is created in, i'm not sure how one escapes via thenables though

Reply:
Neither am I, but it's an interesting thought experiment.

So the context is set on `vm.Module` construction, not on instantiate or link etc? That seems to be what avoids a possible escalation.

Reply:
> default context is the context it is created in

Minor correction: it’s always the top context. 

> So the context is set on `vm.Module` construction, not on instantiate or link etc? That seems to be what avoids a possible escalation.

That, and that there seems to be no reason to allow mixing of contexts in a module graph anyway.